include/fiction/algorithms/physical_design/ortho_ordering_network.hpp

```diff
@@ -0,0 +1,633 @@
+//
+// Created by benjamin on 11.04.23.
+//
+
+#ifndef FICTION_ORTHO_ORDERING_NETWORK_HPP
+#define FICTION_ORTHO_ORDERING_NETWORK_HPP
+
+#include "fiction/algorithms/network_transformation/inverter_substitution.hpp"
+#include "fiction/networks/views/input_ordering_view.hpp"
+#include "orthogonal.hpp"
+
+namespace fiction
+{
+
+namespace detail
+{
+
+template <typename Ntk>
+void paint_node_and_edges(const coloring_container<Ntk>& ctn, mockturtle::node<Ntk> const& n, uint32_t color)
```

Comment:
- please add a docstring
- please do not mix West `const` and `const` East
- `color` can be `const`